config/jobs/cert-manager/cert-manager/release-next/cert-manager-release-next-periodics.yaml

```diff
@@ -1,38 +1,60 @@
 # These tests are configured to only run weekly for now.
-# Why? Because there's no point testing the release-1.8 branch until we do the first 1.8 release: 1.8.0-alpha.0
+# Why? Because there's no point testing the release-1.9 branch until we do the first 1.9 release: 1.9.0-alpha.0
 # See Step 13.3 in https://cert-manager.io/docs/contributing/release-process/
 #
 periodics:
-- name: ci-cert-manager-next-bazel
+- name: ci-cert-manager-next-make-test
   interval: 168h # 1 week
   agent: kubernetes
   decorate: true
   extra_refs:
   - org: cert-manager
     repo: cert-manager
-    base_ref: release-1.8
+    base_ref: release-1.9
```

Comment:
Suggestion: as above, no release-1.9 branch. If we have periodics running against master these tests aren't useful until we release v1.9.0-alpha.0

Reply:
I wasn't able to "deactivate" those jobs and wasn't happy with commenting them out, is it OK if I put an enormous interval of 6 months with a clear explanation at the top? I have done that in e21fbac.

Reply:
I have removed all of the jobs but kept the file. The file starts with a comment that explains why the jobs are gone for now.